docs/concepts.rst

```diff
@@ -91,7 +91,7 @@ time by various "Committers" across any number of "Branches". Though there are
 many conceptual similarities in what a Git repo and a Hangar Repository
 achieve, Hangar is designed with the express purpose of dealing with numeric
 data. As such, when you read/write to/from a Repository, the main way of
-interaction with information will be through (an arbitrary number of) Arraysets
+interaction with information will be through (an arbitrary number of) Columns
 in each Commit. A simple key/value store is also included to store metadata,
 but as it is a minor point is will largely be ignored for the rest of this
```

Comment:
May be mention that this metadata is commit level and will not be part of the history.

Reply:
This will be in a seperate PR.